docker/freebsd.sh

```diff
@@ -8,6 +8,47 @@ set -euo pipefail
 # shellcheck disable=SC1091
 . lib.sh
 
+# we prefer those closer in geography to the US. they're triaged in
+# order of ease of use, reliability, and then geography. the mirror
+# list is at https://docs.freebsd.org/en/books/handbook/mirrors/.
+# these mirrors were known to work as of 2022-11-27. this does
+# not include any mirrors that are known to be rate-limited or
+# commercial. everything returns HTML output.
+MIRRORS=(
+    # this is a guaranteed mirror, unlike those below.
+    "http://ftp.freebsd.org/pub/FreeBSD/releases"
```

Comment:
The main mirror got changed `curl --list-only ftp.freebsd.org/pub/FreeBSD/releases/amd64` no longer lists the releases, so we have to use our HTML regex for everything.